tests/conftest.py

```diff
@@ -53,7 +53,7 @@ class SubmitterClient:
     """Class for submitter in the AWS lambda environment"""
 
     def crash_id_to_key(self, crash_id):
-        return "v1/processed_crash/%s" % crash_id
+        return "v2/raw_crash/%s/%s/%s" % (crash_id[0:3], crash_id[-6:], crash_id)
```

Comment:
When we switch to GCP, we can nix the entropy part and just do `v2/raw_crash/DATE/CRASHID`. That'll be nice.